<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="checkBoxApple">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Apple</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOrange">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Orange</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Q1</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditQuestion1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOption1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Steve Jobs</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOption2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bill Gates</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Q1. who is the founder of appleï¼Ÿ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>120</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Q2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Q3</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Submit</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
